aardvark/manage.py

```diff
@@ -69,8 +56,13 @@ def run(self):
             if not ACCOUNT_QUEUE.empty():
```

Comment:
Spin semantic isn't great. But this runs only during the duration of updates. Not sure if work to change it would bring enough benefit

Reply:
Will be addressed by #116